interbtc-spec/docs/source/spec/security.rst

```diff
@@ -83,26 +77,29 @@ Scalars
 ParachainStatus
 ...............
 
-Integer/Enum (see ``StatusCode`` below). Defines the current state of the BTC Parachain. 
+Stores the status code (:ref:`statusCode`) which defines the current state of the BTC Parachain. 
 
 
 Errors
-........
+......
 
-Set of error codes (``ErrorCode`` enums), indicating the reason for the error. The ``ErrorCode`` entries included in this set specify how to react to the failure.
+Stores the set of error codes (:ref:`errorCode`), indicating the reason for the error.
```

Comment:
There is no longer any reason to store a set of errors, I suggest we change this in the implementation. cc: @nud3l @sander2

Reply:
Let's keep it that way just in case we might ever want to include new error codes. That way we would not have to change the APIs